frontend/pages/schedule/ManageSchedulePage/ManageSchedulePage.tsx

```diff
@@ -119,8 +116,13 @@ const ManageSchedulePage = ({
   router,
 }: ITeamSchedulesPageProps): JSX.Element => {
   const { renderFlash } = useContext(NotificationContext);
-  const { MANAGE_PACKS, MANAGE_SCHEDULE, MANAGE_TEAM_SCHEDULE } = paths;
-  const handleAdvanced = () => router.push(MANAGE_PACKS);
+  const {
+    // MANAGE_PACKS, // Packs UI removed
+    MANAGE_SCHEDULE,
+    MANAGE_TEAM_SCHEDULE,
+  } = paths;
+
+  // const handleAdvanced = () => router.push(MANAGE_PACKS);  // Packs UI removed
```

Comment:
curious why you commented out this instead of removing it.

Reply:
All the components showing the packs is commented out, if I were to remove dependencies, I should just remove that code as well, but I was asked just to comment out code.